io.github.qnapi.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.qnapi
+default-branch: stable
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: "5.10"
```

Comment:
That's a very old runtime, mind trying building it with a newer one?

Reply:
Unfortunately, it fails on anything newer. So I thought flatpak maybe an option to run on newer distro without a problem.

Reply:
What makes it fail ?

Reply:
```
diff --git a/gui/src/forms/subdatawidget.h b/gui/src/forms/subdatawidget.h
index f6f1129..103f25c 100644
--- a/gui/src/forms/subdatawidget.h
+++ b/gui/src/forms/subdatawidget.h
@@ -15,6 +15,7 @@
 #ifndef SUBDATAWIDGET_H
 #define SUBDATAWIDGET_H
 
+#include <QIcon>
 #include <QWidget>
 #include "ui_subDataWidget.h"
 ```

This should be enough to build it against 5.14. That being said, if upstream is completely dead, I'm not convinced if it's a good fit for Flathub unless you commit to patching it to work with supported runtimes.

Reply:
Thanks. I also found that. I am going to patch this project or even fork in the future.

Reply:
If you plan to fork it, it has to happen before submitting the app here.

Reply:
I'd like to first try to make some improvements like working portals and remove the dependency of external 7z. Fork without significant improvement is pretty pointless. Maybe after that original author allows give me to write access to repro? For now I can patch it to work witch current runtime.